modules/cpmso-creating-cr.adoc

```diff
@@ -6,4 +6,99 @@
+
+[IMPORTANT]
+====
+For more information about the structure and parameters of the CR, see "Control Plane Machine Set Operator configuration".
+====
+////
+//Experimental, might create a "default" CR??? If so, then they could just do the same thing as AWS upgraded to 4.12
+. Create a `ControlPlaneMachineSet` CR by running the following command:
++
+[source,terminal]
+----
+$ oc --namespace openshift-machine-api edit controlplanemachineset.machine.openshift.io cluster
+----
+//This didn't work on clusterbot but leaving it here until I can chat more with dev
+////
+
+//This should work tho:
+.Procedure
+
+. Create a YAML file using the following template:
++
+--
+.Control plane machine set CR YAML file template
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1
+kind: ControlPlaneMachineSet
+metadata:
+  name: cluster
+  namespace: openshift-machine-api
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <cluster_id> <1>
+      machine.openshift.io/cluster-api-machine-role: master
+      machine.openshift.io/cluster-api-machine-type: master
+  state: Active <2>
+  strategy:
+    type: RollingUpdate <3>
+  template:
+    machineType: machines_v1beta1_machine_openshift_io
+    machines_v1beta1_machine_openshift_io:
+      failureDomains:
+        platform: <platform> <4>
+        <platform_failure_domains> <5>
+      metadata:
+        labels:
+          machine.openshift.io/cluster-api-cluster: <cluster_id> <6>
+          machine.openshift.io/cluster-api-machine-role: master
+          machine.openshift.io/cluster-api-machine-type: master
+      spec:
+        providerSpec:
+          value:
+            <platform_provider_spec> <7>
+----
+<1> Specify the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster. You must specify this value when you create a `ControlPlaneMachineSet` CR. If you have the OpenShift CLI (`oc`) installed, you can obtain the infrastructure ID by running the following command:
++
+[source,terminal]
+----
+$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+----
+<2> Specify the state of the Operator. When the state is `Inactive`, the Operator is not operational. You can activate the Operator by setting the value to `Active`. 
++
+[IMPORTANT]
+====
+Before you activate the Operator, you must ensure that the CR configuration is correct for your cluster requirements.
+====
+<3> Specify the update strategy for the cluster. The allowed values are `OnDelete` and `RollingUpdate`. The default value is `RollingUpdate`. 
+//For more information about update strategies, see "Updating the control plane configuration".
+<4> Specify your cloud provider platform name. The allowed values are `aws`, `azure`, and `vsphere`. 
```

Comment:
These are actually capitalised using Pascal Case (and validation will fail if the casing is incorrect)
```suggestion
<4> Specify your cloud provider platform name. The allowed values are `AWS`, `Azure`, and `VSphere`. 
```

Reply:
Thanks!